### PR TITLE
Add Postype

### PR DESCRIPTION
**Pull request by contextnerror**

### CODE REVIEW

File: entries/p/postype.com.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "Postype": {
+    "domain": "postype.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.postype.com/hc/ko/articles/21876052282009-2%EB%8B%A8%EA%B3%84-%EC%9D%B8%EC%A6%9D-%EC%84%A4%EC%A0%95-%ED%95%B4%EC%A0%9C",
```
Comment (hkamran80) [inline entries/p/postype.com.json]:
```suggestion
    "documentation": "https://help.postype.com/hc/ko/articles/21876052282009",
    "recovery": "https://help.postype.com/hc/ko/articles/21876052282009#01H8NMJNY1KWFB5H5YZ49HBC0J",
```

File: entries/p/postype.com.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "categories": [
+      "entertainment",
+      "social"
```
Comment (hkamran80) [inline entries/p/postype.com.json]:
```suggestion
      "retail"
```

From what I could glean from their [about page](https://www.postype.com/about), it seems to be more of a marketplace.

File: img/p/postype.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><path d="M32 0v32H0V0h32ZM18.744 7H7.34v2.689h2.094V25h2.404V9.689h2.093V25h2.403v-4.803l2.41-.004a6.596 6.596 0 0 0 6.596-6.596A6.596 6.596 0 0 0 18.744 7Zm.246 2.7a3.87 3.87 0 1 1 0 7.74h-2.61V9.7Z" fill-rule="evenodd"/></svg>
```
Comment (hkamran80) [inline img/p/postype.com.svg]:
```suggestion
<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><path d="M32 0v32H0V0h32ZM18.744 7H7.34v2.689h2.094V25h2.404V9.689h2.093V25h2.403v-4.803l2.41-.004a6.596 6.596 0 0 0 6.596-6.596A6.596 6.596 0 0 0 18.744 7Zm.246 2.7a3.87 3.87 0 1 1 0 7.74h-2.61V9.7Z" /></svg>
```

Thank you for minifying the SVG!

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/p/postype.com.json (hunk 2 above)
Comment (contextnerror) [reply entries/p/postype.com.json]: In a sense? It’s more like a patreon thing where you can pay creators to access content. There is a physical store but that’s a recent addition and still beta.

Comment (hkamran80) [reply entries/p/postype.com.json]: In that case, let's go with `crowdfunding` and `retail` as the categories.

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**